packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -366,7 +371,7 @@ const handleShortcutClick = (shortcut: Shortcut) => {
     ? shortcut.value()
     : shortcut.value
   if (shortcutValue) {
-    emit(dayjs(shortcutValue).locale(lang.value))
+    emit(dayjs(shortcutValue).locale(lang.value), true)
```

Comment:
You seem to have made a lot of unnecessary modifications, maybe you only need to define an `isShortcut` variable externally, and then assign a value of `true` here, and there is no need to modify other places.

Reply:
Modified, thank you beta.